samples/todo-app/frontend/server.js

```diff
@@ -14,26 +12,15 @@ app.prepare().then(() => {
     const server = express();
     server.use(bodyParser.json());
 
-    const mongo = mongodb.MongoClient.connect(process.env.MONGO_CONNECTION_STRING, {
-        useNewUrlParser: true,
-        useUnifiedTopology: true
-    });
-    mongo.then(() => console.log("Connected to Mongo server"));
-    mongo.catch(reason => {
-        console.error(reason);
-        process.exit(1);
-    });
-
     server.get("/api/todos", function (req, res) {
-        mongo.then(client => {
-            const todos = client.db("todos").collection("todos");
-            todos.find({}).toArray((err, docs) => {
-                if (err) {
-                    res.status(500).send(err);
-                } else {
-                    res.send(docs);
-                }
-            });
+        requestData(req, process.env.DATABASE_API_SERVICE_HOST, process.env.DATABASE_API_SERVICE_PORT, '/todos', 'GET', null, (data, error) => {
```

Comment:
I'm not able to see where this env var is getting set?

Reply:
It's set automatically by Kubernetes itself, cf. https://kubernetes.io/docs/concepts/containers/container-environment/#cluster-information